api/projects.types.ts

```diff
@@ -5,9 +5,9 @@ export enum ProjectLanguage {
 }
 
 export enum ProjectStatus {
-  stable = "stable",
+  stable = "info",
   warning = "warning",
-  critical = "critical",
+  critical = "error",
```

Comment:
Great job. Exactly the way to go. I'd just rename the keys as well as they reflect the API values. In the component you then just map the status returned by the API to a text that is displayed.